server/utils/helpers.ts

```diff
@@ -20,8 +20,11 @@ import {
   mapKeys,
   mapValues,
   snakeCase,
+  get,
```

Comment:
Seems `get` is not used in this file. Remove it?

Reply:
Done